loader/src/model.js

```diff
@@ -29,9 +29,13 @@ const LocationType = {
  * @enum {string}
  */
 const VaccineProduct = {
-  pfizer: "pfizer",
-  moderna: "moderna",
+  astraZeneca: "astra_zeneca",
   janssen: "jj",
+  moderna: "moderna",
+  novavax: "novavax",
+  pfizer: "pfizer",
+  pfizerAge5_11: "pfizer_age_5_11",
+  pfizerAge2_4: "pfizer_age_2_4",
```

Comment:
Will happily take feedback on these names.

Reply:
FYI @cmroberts if you are working on HyVee, you’ll probably want to use these new enum values.

Reply:
Noted! Working on those two loaders today